src/app/pages/things/opcua/opcua.component.ts

```diff
@@ -39,6 +40,12 @@ export class OpcuaComponent implements OnInit {
         title: 'Name',
         addable: true,
         filter: false,
+        valuePrepareFunction: (cell) => {
```

Comment:
From the documentation: 
```This function will be invoked with 2 parameters: cell, row```
Maybe add row param, as we do in other cases where we don't use extra parameters.